components/scream/src/share/io/tests/scorpio_tests.cpp

```diff
@@ -132,44 +136,44 @@ TEST_CASE("scorpio_interface_output", "") {
   get_dof(ekat::size(x_data), myrank, numranks, dof_x[0], xstart,xstop);
   std::vector<Int> x_dof(dof_x[0]);
   for (int ii=xstart,cnt=0;ii<=xstop;++ii,++cnt) {
-    x_dof[cnt] = ii+1;  // Add one to index since C++ starts at 0 and Fortran starts at 1
+    x_dof[cnt] = ii;  // Add one to index since C++ starts at 0 and Fortran starts at 1
```

Comment:
Is the comment still relevant?

Reply:
Actually, scorpio_tests will be deleted.  I kept it around for reference, but given that this PR is ready to merge I have deleted it in the most recent version.